Subject: Check for broken command in system crontab
Last-Update: 2010-04-11

This patch apparently adds a test for a superfluous star in a system crontab
entry, eg:

    * * * * * * some_command

(Notice the 6th star, where only 5 should be). Otherwise, this patch doesn't
make sense. Not that it makes much sense this way, either...

NOT Acked
Index: patched/entry.c
===================================================================
--- patched.orig/entry.c	2010-05-06 18:18:35.043561126 +0200
+++ patched/entry.c	2010-05-06 18:18:35.745561147 +0200
@@ -246,6 +246,9 @@
 			goto eof;
 		}
 		Debug(DPARS, ("load_entry()...uid %d, gid %d\n",e->uid,e->gid))
+	} else if (ch == '*') {
+		ecode = e_cmd;
+		goto eof;
 	}
 
 	e->uid = pw->pw_uid;
